frontend/src/metabase/querying/components/FieldPanel/types.ts

```diff
@@ -4,6 +4,7 @@ export type ColumnItem = {
   column: Lib.ColumnMetadata;
   displayName: string;
   isSelected: boolean;
+  isEditable: boolean;
```

Comment:
Presence of `isEditable` is confusing given that `isDisabled` is already here (makes me think and need to check how are they different).
Since `isEditable` is always just a negation of `isDisabled`, it's simpler to not have it.

```suggestion
```

Reply:
Fixed